src/openstudio_lib/DesignDayGridView.cpp

```diff
@@ -57,41 +57,41 @@
 // These defines provide a common area for field display names
 // used on column headers, and other grid widgets
 
-#define NAME "Design Day Name"
-#define SELECTED "All"
+#define NAME openstudio::DesignDayGridView::tr("Design Day Name")
```

Comment:
Why can't you just do:

`#define NAME tr("Design Day Name")`

?

Reply:
Because when the macro is expanded, it doesn't know the context of the Q_Object it's doing it for. See https://doc.qt.io/qt-5/i18n-source-translation.html#using-tr-for-all-literal-text